src/Composer/InstalledPackageResolver.php

```diff
@@ -72,4 +73,25 @@ private function createInstalledPackages(array $packages): array
 
         return $installedPackages;
     }
+
+    private function resolveVendorDir(): string
+    {
+        $projectComposerJsonFilePath = \getcwd() . '/composer.json';
```

Comment:
this should can use `$this->projectDirectory` 

```suggestion
        $projectComposerJsonFilePath = $this->projectDirectory . '/composer.json';
```

Reply:
Indeed! My bad